app/views/ProjectEdit/Connector/ConnectorDetail/index.tsx

```diff
@@ -251,21 +348,17 @@ function ConnectorDetail(props: Props) {
         return lastFetchedDates[0];
     }, [connector]);
 
-    const [
-        editConnectorModalShown,
-        showEditConnectorModal,
-        hideEditConnectorModal,
-    ] = useModalState(false);
+    const isProcessing = useMemo(() => (
+        connector?.sources?.some((source) => source.status === 'PROCESSING')
+    ), [connector]);
```

Comment:
Do we want to disable retrigger if "some source is processing" or "all are processing"?
@AdityaKhatri @thenav56